src/screens/Learn/Glossary/Glossary.tsx

```diff
@@ -16,6 +16,22 @@ import { Anchor } from 'components/TableOfContents';
 import { formatMetatagDate, formatNumericalDate } from 'common/utils';
 import ScrollToTopButton from 'components/SharedComponents/ScrollToTopButton';
 import { useScrollToTopButton } from 'common/hooks';
+import Footer from 'screens/Learn/Footer/Footer';
+import ExternalLink from 'components/ExternalLink';
+
+function getGlossaryFooter(): React.ReactElement {
+  return (
+    <Fragment>
+      *Created in collaboration with COVID Explained, a team of researchers from
+      Brown University, Harvard Medical School, Massachusetts Institute of
+      Technology and Massachusetts General Hospital. Learn more at{' '}
+      <ExternalLink href="https://explaincovid.org/about">
+        {' '}
```

Comment:
Do we need this extra space?

Reply:
oo nope i think the linter added that at some point. removing